setup.cfg

```diff
@@ -12,3 +12,28 @@ versionfile_source = provenance/_version.py
 versionfile_build = provenance/_version.py
 tag_prefix =
 parentdir_prefix = provenance-
+
+
+[flake8]
+exclude = docs
+ignore = E203,E266,E501,W503,E722,E402,C901,E731,E122
+max-line-length = 100
+max-complexity = 18
+select = B,C,E,F,W,T4,B9
+
+[yapf]
+based_on_style = pep8
+spaces_before_comment = 4
+split_before_logical_operator = true
+
```

Comment:
Let me know if you have any suggestions on what settings we should use as defaults